pkg/kubelet/eviction/defaults_windows.go

```diff
@@ -0,0 +1,26 @@
+//go:build windows
+// +build windows
+
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package eviction
+
+// DefaultEvictionHard includes default options for hard eviction.
+var DefaultEvictionHard = map[string]string{
+	"memory.available": "500Mi",
+	"nodefs.available": "10%",
```

Comment:
Can kubernetes in windows support a separate disk? If so, we should probably add imagefs.available as a default option?

I guess inodes are linux only so only that should be needed.

Reply:
I need to check if Windows supports separate disks or not - I don't remember :-/

Reply:
ah - good to know!

Reply:
It does look like the kubelet flag `--root-dir` works on windows and can target a directory on another drive.
Let me add the `image.fs.available` default to the windows defaults.
Thanks @kannon92

Reply:
So its not a kubelet setting actually. The container runtime is allowed to say I want my images/containers stored on a separate disk from kubelet. So I'd look into windows containers (I admit I know nothing about this area so I don't even know what that is).

I'm actually working on a blog post around this area: https://github.com/kubernetes/website/pull/43971

The configuration for the separate filesystem comes from storage configurations for crio/containerd actually. Kubelet doesn't know or care about that.

Reply:
got it. either way i added the default back in for windows :)